packages/blitz-next/src/error-component.tsx

```diff
@@ -31,7 +31,7 @@ export class ErrorComponent<P = {}> extends React.Component<P & ErrorProps> {
   render() {
     const {statusCode} = this.props
     const title = this.props.title || statusCodes[statusCode] || "An unexpected error has occurred"
-
+    const Head = dynamic(() => import("next/head"))
```

Comment:
- this should not be in a render function, should be at top level in the file.
- what affect does this have on legacy apps? I think this will trigger suspense?

Reply:
@flybayer just verified there is no impact on legacy apps